overlayfs-tools/makefile

```diff
@@ -1,12 +1,16 @@
 CFLAGS = -Wall -std=c99
 LFLAGS = -lm
+LIBS = -lfts
 CC = gcc
 
 all: overlay
 
 overlay: main.o logic.o sh.o
 	$(CC) $(LFLAGS) main.o logic.o sh.o -o overlay
 
+external: main.o logic.o sh.o
```

Comment:
I actulay used alpine first but this flag just enables the use of an external lib instead of the fts lib build in to libc. Alpine uses musl instead of libc and musl doesn't have fts integrated so it's an external lib. There might actually be other usecase for this.